sys-apps/iproute2/files/iproute2-6.10.0-fix-implecit-declaration-of-basename-on-musl.patch

```diff
@@ -0,0 +1,24 @@
+Bug: https://bugs.gentoo.org/926341
+From: Brahmajit Das <brahmajit.xyz@gmail.com>
+Date: Sun, 21 Jul 2024 07:31:49 +0000
+Subject: [PATCH 2/2] Fix implecit declaration of basename on musl
+
+On musl there is only one implementation of musl which comes from
+libgen.h, unlike glibc which has two implementations. I'm not really
+sure if using the basename from libgen.h is correct over here i.e. using
+the posix implementation of basename.
```

Comment:
If you are not sure the POSIX implementation is correct here, please look into it and confirm it.